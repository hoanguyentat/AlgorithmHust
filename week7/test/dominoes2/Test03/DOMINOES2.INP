8
7 7 4
1 4
1 3
1 7
1 6
2 3
3 6
5 7
4
2
2
5
4 6 4
4 3
4 4
1 4
1 1
1 1
4 1
1
4
4
1
4 18 2
3 4
4 4
2 3
4 4
3 4
3 2
3 3
2 2
1 2
1 1
2 2
4 2
2 2
4 4
1 2
4 2
1 2
4 2
1
4
17 9 10
8 11
7 8
9 16
17 16
14 9
17 15
13 14
10 6
6 12
14
5
13
14
2
1
7
17
15
4
16 6 11
16 6
3 14
3 5
15 5
4 15
9 13
12
9
3
15
15
6
7
4
2
11
4
6 2 6
2 5
2 1
5
5
2
4
1
1
16 11 8
3 2
2 10
8 9
7 13
10 4
6 2
4 15
7 3
6 13
7 8
7 5
13
11
12
10
16
11
9
16
14 6 1
2 14
1 13
11 7
13 14
12 11
1 4
10