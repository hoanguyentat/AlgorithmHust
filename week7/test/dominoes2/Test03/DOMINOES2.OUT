6
3
4
14
12
4
10
1